packages/fast-usdc/src/exos/advancer.js

```diff
@@ -17,6 +17,12 @@ import {
 } from '../type-guards.js';
 import { makeFeeTools } from '../utils/fees.js';
 
+/**
+ * @file Advancer subscribes (handleTransactionEvent) to events published by the
```

Comment:
`@file` docs don't tend to show up when navigating code. How about putting this on `prepareAdvancer`?

Reply:
a 1-line `/** @file main export: @see {prepareAdvancer} */` at the top could help when looking at the file itself.

Reply:
I think this `@file` comment is good but file-level comments should be at the top of the file

Reply:
done